tests/unittests/cmd/devel/test_hotplug_hook.py

```diff
@@ -244,3 +247,99 @@ def test_retry(self, mocks):
             call(10),
             call(30),
         ]
+
+
+@pytest.mark.usefixtures("fake_filesystem")
+class TestEnableHotplug:
+    @mock.patch(M_PATH + "util.write_file")
+    @mock.patch(
+        M_PATH + "util.read_hotplug_enabled_file",
+        return_value={"scopes": []},
+    )
+    @mock.patch(M_PATH + "install_hotplug")
+    def test_enabling(
+        self,
+        m_install_hotplug,
+        m_read_hotplug_enabled_file,
+        m_write_file,
+        mocks,
+    ):
+        mocks.m_init.datasource.get_supported_events.return_value = {
+            EventScope.NETWORK: {EventType.HOTPLUG}
+        }
+
+        enable_hotplug(mocks.m_init, "net")
+
+        assert [
+            call([EventType.HOTPLUG])
+        ] == mocks.m_init.datasource.get_supported_events.call_args_list
+        assert [call()] == m_read_hotplug_enabled_file.call_args_list
+        assert [
+            call(
+                settings.HOTPLUG_ENABLED_FILE,
+                '{"scopes": ["network"]}',
+                omode="w",
+                mode="640",
```

Comment:
Mode should be an int.

```suggestion
                mode=0o640,
```

Reply:
Good catch here, thanks!